updater/dep.go

```diff
@@ -0,0 +1,29 @@
+package updater
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/shurcooL/Go-Package-Store"
+)
+
+// Dep is an Updater that updates Go packages in a project managed by dep.
+//
+// It requires the dep binary to be available in PATH.
+type Dep struct {
+	// Dir specifies where the dep binary is executed.
+	// If empty, current working directory is used.
+	Dir string
+}
+
+func (d Dep) Update(repo *gps.Repo) error {
```

Comment:
Ignoring because this implements an interface, and I don't want to copy/paste the interface documentation into multiple places, causing it to more easily fall out of date.

I could write a `// Update implements Updater interface.`, but it doesn't seem worth it at this point.